pom.xml

```diff
@@ -452,6 +454,20 @@
             <version>${version.com.squareup.okhttp3.mockwebserver}</version>
             <scope>test</scope>
         </dependency>
+
```

Comment:
OK, I didn't see that, but considering that JCache integration can never be correct, I think we should drop it anyway (as previously discussed).